app/build.gradle

```diff
@@ -39,7 +39,10 @@ android {
 
 dependencies {
     compile project(":flexbox")
+    compile "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
     compile "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    compile "com.android.support:preference-v7:${rootProject.ext.supportLibVersion}"
+    compile "com.android.support:preference-v14:${rootProject.ext.supportLibVersion}"
```

Comment:
This is a nice to have to keep the material theme on API level 14+.

See http://stackoverflow.com/questions/32070670/preferencefragmentcompat-requires-preferencetheme-to-be-set

Added explanation to the values-v14/styles.xml
https://github.com/google/flexbox-layout/pull/45/files#diff-b7d8affb5986661f0b964437076193dbR20